extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/CalciteSelectQueryMSQTest.java

```diff
@@ -183,4 +190,53 @@ public void testArrayAggQueryOnComplexDatatypes()
       );
     }
   }
+
+  @Test
+  public void testMsqDenySelectEnabledQuery()
+  {
+    msqCompatible();
```

Comment:
These tests probably want to be in `CalciteInsertDmlTest`, if they can share config. Those tests are run only for MSQ. The test in this form runs for both MSQ and non-MSQ, which may not be what you wanted.